Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.ts

```diff
@@ -86,6 +95,31 @@ export class UserDetailComponent implements OnInit, OnDestroy {
     );
   }
 
+  public openInformationIncorrectDialog(): void {
+    const wrapper = document.getElementsByClassName('pb-margin-20')[0];
+    wrapper.classList.add('blur');
+    const name = {
+      firstNames: this.person.Firstname,
+      lastNames: this.person.Surname,
+      Titel: this.person.Title,
+      Id: this.person.Id
+    };
+    const dialogConfig : MatDialogConfig = {
+      autoFocus : true,
+      height : '23vh',
+      width : '30vw',
```

Comment:
Not responsive: 
![image](https://user-images.githubusercontent.com/39745446/67388463-fd004c00-f598-11e9-95af-227ea05bf6e5.png)

Maybe you can use `pb-margin-left-right-responsive`

Reply:
Just remove the width and height attribute for opening the dialog it should automatically scale